DUI3-DX/Sdk/Speckle.Converters.Common.DependencyInjection/ToHost/ConverterWithFallback.cs

```diff
@@ -54,24 +52,13 @@ public object Convert(Base target)
       return FallbackToDisplayValue(displayValue);
```

Comment:
Need some thoughts on it -> How can we report fallback conversion, or should we?

Reply:
See other comment above, Perhaps we don't need to, since we will know the original and result type and that's the main thing users will care about.

But if we wanted to go further, and know the converter that was used, I have an idea, but I think we should wait till we can get some feedback from the team.